.travis.yml

```
@@ -95,6 +95,7 @@ jobs:
         - export CONDA_PATH="\$env:Path = ';C:\miniconda3;C:\miniconda3\Library\mingw-w64\bin;C:\miniconda3\Library\usr\bin;C:\miniconda3\Library\bin;C:\miniconda3\Scripts;C:\miniconda3\bin;C:\miniconda3\condabin;C:\iverilog\bin;' + \$env:Path"
         - powershell "$CONDA_PATH; conda install --yes -c msys2 m2-base m2-make m2w64-toolchain libpython"
         - powershell "$CONDA_PATH; conda install --yes virtualenv"
+        - powershell '$CONDA_PATH; cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\'
```

I think this needs a todo comment pointing at this PR. I'm happy to commit the hack in the interest of having coverage, but lets leave a breadcrumb in case someone works out how to remove it.

```suggestion
        # needed to make the builtin sqlite3 module (used by coverage) work - see gh-1909
        - powershell '$CONDA_PATH; cp C:\miniconda3\Library\bin\sqlite3.dll C:\miniconda3\DLLs\'
```

---

Because we run with coverage on in tox, this change will effectively prevent any Windows user from running regressions. I'll move it into tox.ini (along with the message) first.